kinsumer.go

```diff
@@ -483,6 +484,51 @@ func (k *Kinsumer) CreateRequiredTables() error {
 	return g.Wait()
 }
 
+func (k *Kinsumer) ResetCheckpoints(sess *session.Session) error {
+	entries, err := k.dynamodb.Scan(&dynamodb.ScanInput{TableName: aws.String(k.checkpointTableName)})
+	if err != nil {
+		return err
+	}
+
+	for _, entry := range entries.Items {
+		shard := entry["Shard"].S
+
+		if shard == nil {
+			return fmt.Errorf("found %v for Shard entry", entry["Shard"].S)
```

Comment:
Probably fine to use "%s" here and then let the AttributeValue make it pretty https://github.com/aws/aws-sdk-go/blob/master/service/dynamodb/api.go#L6125-L6127